packages/web-wallet/src/components/Wallet/AddFunds/AddFunds.js

```diff
@@ -36,17 +36,15 @@ class AddFunds extends Component {
     };
   }
 
-  componentDidMount() {
-    this.resetPathPayment();
-  }
-
-  resetPathPayment = () => {
-    this.props.resetRequest('pathPayment');
+  resetForm = () => {
+    this.setState({ delta: 0 });
   };
 
   getAssetName = () =>
     this.props.match.params.asset === 'native' ? 'XLM' : 'MOBI';
 
+  onTabChange = index => (this.currentTabIndex = index);
```

Comment:
It seems like we don't need to store current tab Index here, completely. Let's get rid of it.

Reply:
@lysyi3m check out my comment in the outdated change above